src/trajectory.jl

```diff
@@ -233,6 +260,14 @@ function transition(
     return Transition(z, tstat)
 end
 
+function transition(rng, h, τ::Trajectory{I, <:FixedLength}, ::Type{TS}, z) where {I, TS}
+    @unpack integrator, criterion = τ
+    # Create the corresponding `FixedNSteps` criterion
+    L = max(1, floor(Int, criterion.λ / nom_step_size(integrator)))
```

Comment:
We would need to do this for each transition because `L` is step size dependent, e.g. for HMCDA.
If we were to implement this when reconstructing trajectory, the same computation is needed.
Also I think this computaitn is negligible.

Reply:
Thanks for the clarification!